homeassistant/components/wsdot/manifest.json

```diff
@@ -2,9 +2,12 @@
   "domain": "wsdot",
   "name": "Washington State Department of Transportation (WSDOT)",
   "codeowners": [],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/wsdot",
+  "integration_type": "hub",
   "iot_class": "cloud_polling",
   "loggers": ["wsdot"],
   "quality_scale": "legacy",
-  "requirements": ["wsdot==0.0.1"]
+  "requirements": ["wsdot==0.0.1"],
+  "single_config_entry": true
```

Comment:
Why only a single config entry?

Reply:
It didn't make sense to me to have multiple `wsdot` api keys, which is the only thing that the top-level integration manages

Reply:
I think we can allow it if someone has the use case for it, generally if there is no technical reason, it's an easy change to allow it